packages/common/src/stores/ConfigStore.ts

```diff
@@ -10,11 +10,11 @@ type ConfigState = {
   config: Config;
   accessModel: AccessModel;
   settings: Settings;
-  integrationType: IntegrationType | null;
+  integrationType: string | null;
```

Comment:
You're doing this to make it more generic?

Reply:
Yes, by limiting the type to `IntegrationType` the functions can't set other strings than 'JWP' or 'CLEENG'.